redhat-developer/app-services-cli
# feat(kafka): add ability to update Kafka instance

Closes #1012 

Verification is only possible by an org admin (@wtrocki).

### Verification Steps

1. Run `export RHOAS_DEV=true` to enable this hidden feature.
1. Log in using an offline token (currently only supported method)
2. Create a Kafka, or skip this if you already have one.
3. Run `rhoas kafka update`. You should get an interactive prompt to select a new owner. Select `sharedaccount_kafka_devexp`.
4. It should update the Kafka instance owner.
5. Run `rhoas kafka update --owner=<TAB>` and it should load the full list of available users to choose from.

### Type of change


### Checklist
